test/relations.test.js

```diff
@@ -34,9 +34,16 @@ describe('relations', function() {
     db = getSchema();
   });
 
+  after(function(done) {
+     // disconnect from this db to avoid too many connection error
+     // due to multiple instance of connection pool
+    db.disconnect();
+    done();
+  });
+
   describe('hasMany', function() {
     before(function(done) {
-      db = getSchema();
+      // db = getSchema();
```

Comment:
@jannyHou: i was testing something actually:
as you can see there is at the same time a db = getSchema(); beforeAll hook at test root, but also one at each sub-describe test section. (this is from the original implementation). Adding only one afterAll hook to disconnect the db actually did not solved the too many connections issue and brought new ones.
It turns out that this needs to be fixed : either one beforeAll / afterAll hook to get/destroy db. Or one per sub-section, and removing the one at root (as in my last fix)

We can go for one per section if you like.